File: Data Structures & Algorithms/majority-element-ii/submission-0.cpp
class Solution {
public:
    vector<int> majorityElement(vector<int>& nums) {
        
        sort( nums.begin() , nums.end());
        unordered_map<int,int> mpp;
        vector<int> ans;
        for( int i=0; i<nums.size(); i++ ) {
            mpp[nums[i]]++;
        }

        for( int i=0; i<nums.size(); i++ ) {
            if( mpp[nums[i]] > nums.size()/3 ) {
                if( ans.empty() || ans.back() != nums[i]) {
                ans.push_back(nums[i]);
                }
            }
        }

        return ans;

    }
};

File: Data Structures & Algorithms/height-checker/submission-0.java
class Solution {
    public int heightChecker(int[] heights) {
        
        int ct = 0;

        int[] ans  = new int[heights.length];

        for( int i=0; i<heights.length; i++ ) {
            ans[i] = heights[i];
        }

        Arrays.sort(ans);

        for( int i=0; i<heights.length; i++ ) {
            if( heights[i] != ans[i] ) {
                ct++;
            }
        }

        return ct;
    }
}